package.json

```diff
@@ -27,10 +27,11 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
-    "@apollo/client": "^3.4.16",
+    "@apollo/client": "^3.5.9",
     "@auth0/auth0-react": "^1.2.0",
     "@headlessui/react": "^1.4.1",
     "@heroicons/react": "^1.0.1",
+    "@recordreplay/playwright": "^1.10.6",
```

Comment:
This was turned into an optional dependency in https://github.com/RecordReplay/devtools/commit/e74aabdd20bf411deb4b686e06149efd34425063, not sure if that is still necessary.

Reply:
I keep having this problem where `npm i` always adds to `package.json` and I keep removing it manually. I briefly discussed this with Josh, and tried changing versions of node/npm but to no avail.